cloud/create-vm-instances-for-geth.yml

```diff
@@ -0,0 +1,75 @@
+- hosts:
+    - localhost
+  vars_files:
+    - ../concent-builder/consts.yml
+    - "{{ deployment_values }}/var.yml"
+  tasks:
+    - block:
+        - fail: msg="Please supply correct "geth_instance" variable. They are only two option{{ ":" }} geth-testnet/geth-mainnet"
+          when: geth_instance != "geth-testnet" and geth_instance != "geth-mainnet"
+
+        - set_fact:
+            disk_type:    "{% if geth_instance == 'geth-testnet' %} 'pd-standard' {% elif geth_instance == 'geth-mainnet'  %} 'pd-ssd' {% else %} '' {% endif %}"
+            machine_type: "{% if geth_instance == 'geth-testnet' %} 'n1-standard-1' {% elif geth_instance == 'geth-mainnet'  %} 'n1-standard-2' {% else %} '' {% endif %}"
+
+        - name:   Check if static IP address for the "{{ geth_instance }}" instance already exist
+          shell:  gcloud compute addresses list    \
+              --project {{ gke.project }}          \
+              | grep {{ geth_instance }}-ip
+          register:      ip_address_result
+          ignore_errors: yes
+
+        - name:   Check if blockchain disk for the "{{ geth_instance }}" instance already exist
+          shell:  gcloud compute disks list                     \
+              --project   {{ gke.project }}                     \
+              | grep {{ geth_instance }}-blockchain-disk
+          register:      disk_result
+          ignore_errors: yes
+
+        - name:  Check if firewall rule that allows TCP traffic exists
+          shell: gcloud compute firewall-rules list    \
+              --format      json                       \
+              --project     {{ gke.project }}          \
+              | grep allow-geth-traffic
```

Comment:
If you're processing `json` using `jq` to extract the value would be much more reliable than `grep`.

This could be done as a separate issue - `grep` is fine as a temporary solution.

Reply:
Create a separate issue https://github.com/golemfactory/concent-deployment/issues/362